### ISSUE TITLE
parse error from file/copy module output

### ISSUE THREAD

**Issue by nafeger** [ansible/ansible#476]
Make a playbook with a copy module in it.  Have the destination be a directory that your user does not have write access to.  The error message will fail with a very difficult to parse stack trace: 

md5sum=c60c9492363855bdc78f2efce82e499a changed=True

fatal: [spredfast.vm] => Traceback (most recent call last):
  File "/Users/nafeger/ws/ansible/lib/ansible/runner/**init**.py", line 617, in _executor
    exec_rc = self._executor_internal(host)
  File "/Users/nafeger/ws/ansible/lib/ansible/runner/__init__.py", line 656, in _executor_internal
    result = self._execute_template(conn, tmp)
  File "/Users/nafeger/ws/ansible/lib/ansible/runner/__init__.py", line 604, in _execute_template
    exec_rc = self._execute_module(conn, tmp, copy_module, args)
  File "/Users/nafeger/ws/ansible/lib/ansible/runner/__init__.py", line 315, in _execute_module
    result1 = utils.parse_json(res)
  File "/Users/nafeger/ws/ansible/lib/ansible/utils.py", line 186, in parse_json
    tokens = shlex.split(data)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shlex.py", line 279, in split
    return list(lex)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shlex.py", line 269, in next
    token = self.get_token()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shlex.py", line 96, in get_token
    raw = self.read_token()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/shlex.py", line 172, in read_token
    raise ValueError, "No closing quotation"
ValueError: No closing quotation

PLAY RECAP ********************\* 

basically it is trying to parse the error response as json and it cannot do so. Error response included below 

cp: cannot create regular file `/Users/ansible/developer.ini': No such file or directory

**Comment by mpdehaan** [comment]
putting this in the 0.5 bucket, thanks.

**Comment by mpdehaan** [comment]
closed per above
